extensions/justwatch-search/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # JustWatch Search Changelog
 
+## [Open in IMDB + Add Link Icons] - {PR_MERGE_DATE}
```

Comment:
Could you just make a small list of the changes (like in your PR text) here instead of only having a title? This makes it better readable on the Raycast website